src/Data/AVL/Key/Relation/StrictOrder.agda

```diff
@@ -0,0 +1,152 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Pointwise lifting of strict orders for keys for AVL trees.
+------------------------------------------------------------------------
+
+open import Relation.Binary
+
+module Data.AVL.Key.Relation.StrictOrder
+       {k r} (Key : Set k) (_<_ : Rel Key r)
+       where
+
+open import Data.AVL.Key Key
+
+open import Level
+open import Data.Empty
+open import Data.Unit using (⊤)
+open import Data.Product
+open import Function
+open import Relation.Nullary
+
+-- An extended strict ordering relation.
+
+infix 4 _<⁺_
+
+_<⁺_ : Key⁺ → Key⁺ → Set r
```

Comment:
What's the reason you're not defining this using `NonStrictToStrict`?

Reply:
That's the original definition from the AVL module. I kept it the same.
Thinking about it:

* I would rather have a positive definition instead of one involving a negation
* The fact that `_<⁺_` reduces to `_<_` in the `([_], [_])` case is also pretty convenient in the proofs later on

Reply:
Okay that makes sense I guess.